packages/dapp/src/components/StoryFeed/StoryDetails.tsx

```diff
@@ -53,7 +57,13 @@ export const StoryDetails: React.FunctionComponent<StoryDetailsProps> = props =>
       </StoryDetailsFullBleedHeader>
       <StoryDetailsContent>
         <StoryDetailsFlex>
-          <StoryLink href={openGraphData.url} target="_blank">
+          <StoryLink
+            onClick={() =>
```

Comment:
can you confirm that this doesn't cause any issues having an href and an onclick handler? Just want to make sure this doesn't stop event propogation or break the link, for example

Reply:
I tested it, I see the GA response and the link is working.